docs/docs/deployment/hosting/locally-edge-proxy.md

```diff
@@ -206,7 +205,7 @@ services:
      target: /app/config.json
   ports:
    - target: 8000
```

Comment:
```suggestion
   - "8000:8000"
```

Reply:
Yeah, I tend to agree but (for some reason) I stuck as close to the previous definition as possible. I'll just update to the shorter form.